exporter/exporter_test.go

```diff
@@ -648,3 +648,38 @@ func TestResourceName(t *testing.T) {
 	})
 	assert.Equal(t, "general_policy_all_users", norm)
 }
+
+func TestImportingGlobalInitScripts(t *testing.T) {
+	defer common.CleanupEnvironment()()
```

Comment:
please rewrite with `qa.HTTPFixturesApply`, so that total runtime of tests in this package is reasonable